pl/lessons/basics/sigils.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: page
+title: Sigile
+category: basics
+order: 10
+lang: pl
+---
+
+Tworzenie i praca z sigilami.
+
+## Spis treści.
+
+- [Czym są sigile](#Czym-są-sigile)
+  - [Listy znaków](#Listy-znaków)
+  - [Wyrażenia regularne](#Wyrażenia-regularne)
+  - [Ciągi znaków](#Ciągi-znaków)
+  - [Listy słów](#Listy-słów)
+- [Tworzenie sigili](#Tworzenie-sigili)
+
+## Czym są sigile
+
+Elixir posiada specjalną składnię pozwalającą na pracę z literałami. Sigil, bo o nim mowa,  rozpoczyna się znakiem 
+tyldy `~`, a po nim następuje pojedyncza litera. Biblioteka standardowa Elixira dostarcza predefiniowane sigile, ale 
+można też stworzyć własne, gdy potrzebujemy w jakiś sposób rozszerzyć możliwości języka. 
+
+Lista dostępnych sigili to:
+
+  - `~C` Tworzy listę znaków **bez uwzględnienia** interpolacji i interpretacji znaków specjalnych.
+  - `~c` Tworzy listę znaków **z uwzględnieniem** interpolacji i interpretacji znaków specjalnych.
+  - `~R` Tworzy wyrażenie regularne **bez uwzględnienia** interpolacji i interpretacji znaków specjalnych.
+  - `~r` Tworzy wyrażenie regularne **z uwzględnieniem** interpolacji i interpretacji znaków specjalnych.
+  - `~S` Tworzy ciąg znaków **bez uwzględnienia** interpolacji i interpretacji znaków specjalnych.
+  - `~s` Tworzy ciąg znaków **z uwzględnieniem** interpolacji i interpretacji znaków specjalnych.
+  - `~W` Tworzy listę słów **bez uwzględnienia** interpolacji i interpretacji znaków specjalnych.
+  - `~w` Tworzy listę słów **z uwzględnieniem** interpolacji i interpretacji znaków specjalnych.
+
+Ogranicznikami dla listy są:
```

Comment:
"Ogranicznikami dla listy są" -> I would rather translate it as: "Lista separatorów". What do you think about this?